ca/ca.go

```diff
@@ -149,6 +150,13 @@ func NewCertificateAuthorityImpl(
 		[]string{"type"})
 	stats.MustRegister(adoptedOrphanCount)
 
+	lintErrorCount := prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Name: "lint_errors",
+			Help: "Number of linting errors encountered during issuance",
```

Comment:
```suggestion
			Help: "Number of issuances that were halted by linting errors",
```

(because this does not count multiple lints for a single issuance, which is good; that's what we want)